### PR TITLE
Add support for CentOS 8

### PR DESCRIPTION
**Pull request by dhoppe**

### CODE REVIEW

**Comment by dhoppe** [comment]
Looks like there is an issue during the import / verification of the new GPG key.

**Comment by lnemsick-simp** [comment]
@dhoppe Any idea when this will be resolved? Would like to see el8 support added!

**Comment by trevor-vaughan** [comment]
I would like this as well!

**Comment by op-ct** [comment]
:+1: 

CentOS 8 support would be great; it's been out for a year already.  Are the GPG failures still an issue?

**Comment by dhoppe** [comment]
I just restarted the jobs and it looks like the problem has solved itself. 
* https://travis-ci.org/github/voxpupuli/puppet-gitlab/jobs/680048254
* https://travis-ci.org/github/voxpupuli/puppet-gitlab/jobs/680048255

Unfortunately we have to wait for Travis CI now, hoping that all jobs are green.

File: .travis.yml [hunk 1]
```diff
@@ -26,59 +26,67 @@ jobs:
...
       services: docker
     - rvm: 2.5.3
       bundler_args: --without development release
-      env: BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_setfile=debian10-64 CHECK=beaker
+      env: BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_setfile=centos8-64 CHECK=beaker
       services: docker
     - rvm: 2.5.3
       bundler_args: --without development release
-      env: BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_setfile=ubuntu1604-64 CHECK=beaker
+      env: BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_setfile=debian8-64 CHECK=beaker
```
Comment (kenyon) [inline .travis.yml:53]: Might as well remove this since Debian 8 is EOL.

**Comment by dhoppe** [comment]
I agree, but this should be handled by a different pull request.

File: .travis.yml (hunk 1 above)
Comment (dhoppe) [reply .travis.yml:53]: I agree, but this should be handled by a different pull request.

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**